front/src/WebRtc/BBBFactory.ts

```diff
@@ -11,7 +11,7 @@ class BBBFactory {
         }
 
         const allowPolicy =
-            "microphone *; " + "camera *; " + "display-capture *; " + "clipboard-read *; " + "clipboard-write *;";
+            "microphone *; camera *; display-capture *; clipboard-read *; clipboard-write *; screen-wake-lock *;";
```

Comment:
Hey @mktn87,

Thanks a lot for the PR.

Quick question: Can't we just use the fullscreen policy here?

```suggestion
            "microphone *; camera *; display-capture *; clipboard-read *; clipboard-write *; screen-wake-lock *; fullscreen *";
```

That would be easier.

Reply:
You are absolutely right 🤦‍♂️. I'll change that.